drape_frontend/drape_engine.cpp

```diff
@@ -554,7 +554,7 @@ void DrapeEngine::UpdateTraffic(traffic::TrafficInfo const & info)
 #endif
 
   df::TrafficSegmentsColoring segmentsColoring;
-  segmentsColoring.insert(make_pair(info.GetMwmId(), info.GetColoring()));
+  segmentsColoring.emplace(info.GetMwmId(), info.GetColoring());
```

Comment:
Is it guaranteed that segmentsColoring doesn't have key equal to info.GetMwmId()?